libflux/flux-core/src/ast/mod.rs

```diff
@@ -764,7 +764,10 @@ fn get_err_monotype(mt: MonoType) -> String {
 }
 fn get_err_parameters(prs: Vec<ParameterType>) -> String {
     for pr in prs {
-        get_err_parameter(pr);
+        let e = get_err_parameter(pr);
```

Comment:
If I'm understanding this change correctly, did this method just not work at all previously? Is there a test for this behavior? This is a pretty good find and I'd like to make sure we keep it fixed.